Modules/readline.c

```diff
@@ -1034,10 +1040,16 @@ on_startup_hook(void)
 static int
 #if defined(_RL_FUNCTION_TYPEDEF)
 on_pre_input_hook(void)
+{
+#elif defined(__APPLE__) && defined(WITH_APPLE_EDITLINE)
+on_pre_input_hook(const char *text, int state)
+{
+  (void)text;
+  (void)state;
```

Comment:
Would it not be easier to use `Py_UNUSED` on the args instead of the (void) trick?